packages/commons/src/nfc-reader.class.ts

```diff
@@ -0,0 +1,19 @@
+import NFCTag from './nfc-tag.class';
+
+export default class NFCReader {
+  private static instance: NFCReader;
+
+  currentTag: NFCTag | undefined;
```

Comment:
We should use `null` here to make clear this key will always be there, but sometimes it will explicitly be `null` / no set. [See null vs undefined](https://basarat.gitbook.io/typescript/recap/null-undefined)

FYI: If you want to allow undefined you could also write `currentTag?: NFCTag;`. [See optional properties](https://www.typescriptlang.org/docs/handbook/interfaces.html#optional-properties)

```suggestion
  currentTag: NFCTag | null;
```

Reply:
resolved by 7f6fca830e15635acf95b520a301f99af5847662